setup/gcp/installer/util/utils.py

```diff
@@ -19,9 +19,11 @@
 
```

Comment:
File level comment: Arguably this should be something util related and not utils.py. It's already in the utils module. Can we give a more specific name?

Reply:
I think that you can pull out the merge stuff since it's big enough into `merge_util.py`.  Or for a more cooler name, even `merge_engine.py`.